src/htcdaskgateway/cluster.py

```diff
@@ -81,8 +93,8 @@ def scale_batch_workers(self, n):
         condor_logdir = f"{tmproot}/condor"
         credentials_dir = f"{tmproot}/dask-credentials"
         worker_space_dir = f"{tmproot}/dask-worker-space"
-        
-        image_name = f"/cvmfs/unpacked.cern.ch/registry.hub.docker.com/" + self.condor_image
+
+        image_name = "/cvmfs/unpacked.cern.ch/" + self.image_prefix + "/" + self.condor_image
```

Comment:
Should we use f-strings here (and everywhere)?   Is it cleaner to read
```image_name = f"/cvmfs/unpacked.cern.ch/{self.image_prefix}/{self.condor_image}"``` ?

Reply:
I am honestly impartial, I typically don't use f'strings but I have no strong feelings either way